pycbc/scheme.py

```diff
@@ -33,7 +33,7 @@
 from .libutils import get_ctypes_library
 
 try:
-    _libgomp = get_ctypes_library("gomp", ['gomp'])
+    _libgomp = get_ctypes_library("gomp", ['gomp'], mode=ctypes.RTLD_GLOBAL)
```

Comment:
@spxiwh do you think it is worth also trying to move the entire try/except block somewhere inside the CPUScheme class, as I suggested on Slack? In this way we reduce the amount of stuff that needs to happen at import time.